cws-core/src/main/java/jpl/cws/core/db/SchedulerDbService.java

```diff
@@ -605,6 +625,44 @@ public void deleteProcessDefinition(String procDefKey) {
 				"where proc_def_key=?",
 				new Object[] {procDefKey});
 	}
+
+
+	/**
+	 *
+	 */
+	public void deleteAbandonedWorkers(String workerId) {
+		jdbcTemplate.update(
```

Comment:
Please add some comments above each of these SQL statements, to describe in a human-readable way, what they are doing